src/middleware/auth.js

```diff
@@ -0,0 +1,32 @@
+const { roles } = require('../roles');
```

Comment:
Well done bro, you have done a great job, but can we have another middleware to check if the user has been verified?